docker/test/develop.yml

```diff
@@ -0,0 +1,42 @@
+# This needs at least compose 1.6.0
+version: '2'
+
+services:
+  postgresdata:
+    image: postgres:9.5
+    command: /bin/true
+
+  postgres:
+    image: postgres:9.5
+    volumes_from:
+      - postgresdata
+    environment:
+      POSTGRES_PASSWORD: myhuginnpassword
+      POSTGRES_USER: huginn
+
+  huginn_test_postgres:
```

Comment:
Good idea. What do you think about renaming the compose file to `test`? I imagine we can add `develop` config which starts Huginn, the background processes and `guard`.